iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -809,8 +810,14 @@ public void testCreateTableWithColumnComments() {
     for (int i = 0; i < icebergTable.schema().columns().size(); i++) {
       Types.NestedField field = icebergTable.schema().columns().get(i);
       Assert.assertArrayEquals(new Object[] {field.name(), HiveSchemaUtil.convert(field.type()).getTypeName(),
-          field.doc() != null ? field.doc() : "from deserializer"}, rows.get(i));
+          field.doc() != null ? field.doc() : ""}, rows.get(i));
     }
+
+    // Check the columns directly
+    List<FieldSchema> cols = shell.metastore()
+        .run(client -> client.getTable(new GetTableRequest("default", "comment_table")))
```

Comment:
Shall we refactor`shell.metastore().getTable()` to use this implementation too, if this is superior?

Reply:
Changed to use the `shell.metastore().getTable(...)`, and fixed the usage there so we use the new one instead of the deprecated method